migrate-database.php

```diff
@@ -0,0 +1,88 @@
+<?php
+	$debug = false;
+
+	$configuration = parse_ini_file("config.ini", true);
+
+	// ** MySQL settings - You can get this info from your web host ** //
+	$db_configuration = $configuration['database'];
+	/** The name of the database for WordPress */
+	define('DB_NAME', $db_configuration['name']);
+
+	/** MySQL database username */
+	define('DB_USER', $db_configuration['user']);
+
+	/** MySQL database password */
+	define('DB_PASSWORD', $db_configuration['password']);
+
+	/** MySQL hostname */
+	define('DB_HOST', $db_configuration['host']);
+
+	/** Database Charset to use in creating database tables. */
+	define('DB_CHARSET', $db_configuration['charset']);
+
+	/** The Database Collate type. Don't change this if in doubt. */
+	define('DB_COLLATE', $db_configuration['collate']);
+
+	$table_prefix = 'wp_';
+
+	echo "starting migration";
+	define('REPL_DOMAIN_OLD', "vmkrcmar21.informatik.tu-muenchen.de");
+	define('REPL_DOMAIN_NEW', "integreat");
+	define('REPL_PATH_OLD', "/wordpress/");
```

Comment:
Did you add this for further Migrations?

Reply:
Wouldn't it be better, if we allow the user to define this in the config.ini? Now we have to change the file everytime we use this script

Reply:
Well, the idea is to adopt this script for each run.

Reply:
Okay, nevermind then